### PR TITLE
Add Peercoin support

### PR DESCRIPTION
**Pull request by Har01d**
This PR proposes adding Peercoin to 3xpl.

Distinctive features: PoS blocks and a non-standard number of decimals (6 instead of 8)

### CODE REVIEW

**Comment by Har01d** [comment]
The explorer is available at https://3xpl.com/peercoin

**Comment by selfwithin** [comment]
Peercoin Wallet Balance Data are wrong..

**Comment by Har01d** [comment]
There's indeed an issue with some balances, I'm working on a fix.

**Comment by Har01d** [comment]
We're currently repopulating the database.

**Comment by Har01d** [comment]
It seems like the fix has helped with wrong balances.

File: Modules/Common/UTXOTraits.php [hunk 1]
```diff
@@ -163,9 +165,20 @@ private static function bc_hexdec($num)
...
+        if ($value === '0.00000000') return '0';
+        if (strlen(explode('.', $value)[1]) !== 8) throw new ModuleError("satoshi(value: ({$value})): incorrect value");
+        return ltrim(str_replace('.', '', $value), '0');
+    }
+    else
+    {
+        $decimals = $module->currency_details['decimals'];
+        if ($value === '0.' . str_repeat('0', $decimals)) return '0';
+        $value_parts = explode('.', $value);
+        if (strlen($value_parts[1]) !== $decimals) throw new ModuleError("satoshi(value: ({$value})): incorrect number of decimals");
```
Comment (alexqrid) [inline Modules/Common/UTXOTraits.php:181]: For the history: error throwing kept to catch the case and be aware when this happens

> [The Peercoin’s system is the equivalent of allowing an unlimited number of decimals](https://talk.peercoin.net/t/why-was-ppc-designed-to-be-inflationary/316/8)

**Review by alexqrid — APPROVED** [review]
LGTM!

**Comment by Har01d** [comment]
Thanks for the review. The explorer has been running in production (https://3xpl.com/peercoin) for a few days, everything seems to be correct.

**Pull request merged**